moninag/tests/unittests/contact/test_views.py

```diff
@@ -0,0 +1,203 @@
+import json
+
+from django.contrib.auth import authenticate
+from django.core.urlresolvers import reverse
+from django.test import Client
+from django.test import TestCase
+
+from contact.models import Contact
+from registration.models import CustomUser
+
+
+def to_dict(contact):
+
+    return {
+        'id': contact.id,
+        'first_name': contact.first_name,
+        'second_name': contact.second_name,
+        'email': contact.email
+    }
+
+
+class ContactViewTest(TestCase):
+    def setUp(self):
+
+        self.user = CustomUser.objects.create(
+            id=1,
+            first_name='Test',
+            second_name='User',
+            email='test@gmail.com',
+            is_active=True,
+        )
+
+        self.user.set_password('password')
+        self.user.save()
+        self.user = authenticate(username='test@gmail.com', password='password')
```

Comment:
Do you really need this line? :slightly_smiling_face: 
If don't, delete import too :upside_down_face: